packages/reporting-server/README.md

```diff
@@ -98,6 +238,42 @@ for more information, see https://tortoise-orm.readthedocs.io/en/latest/database
 npm run test
 ```
 
+### Running migration tests locally
+
+**Short Version:**  
+To test the migrations on your local machine you can run the following command:
+
+```bash
+npm run test:migrations
```

Comment:
Don't know if it is possible, can we use the same commands in CI as well?

Reply:
No, we cannot because in the localhost we are building a container and the ci is the container itself